app/views/services/deployments/index.html.haml

```diff
@@ -19,4 +19,8 @@
         = t '.actions'
 
   %tbody
+    - if params[:page].to_i > 1
```

Comment:
how about using a gem to do this? Something like [Pagy](https://github.com/ddnexus/pagy)